include/aikido/constraint/TSR.hpp

```diff
@@ -57,7 +58,8 @@ class TSR : public Sampleable,
   TSR(const Eigen::Isometry3d& _T0_w = Eigen::Isometry3d::Identity(),
       const Eigen::Matrix<double, 6, 2>& _Bw =
           Eigen::Matrix<double, 6, 2>::Zero(),
-      const Eigen::Isometry3d& _Tw_e = Eigen::Isometry3d::Identity());
+      const Eigen::Isometry3d& _Tw_e = Eigen::Isometry3d::Identity(),
+      const double _testableTolerance = 1e-6);
```

Comment:
Don't mark arguments that are passed by value as `const` in function declarations.